Purchases/SwiftSources/LocalReceiptParsing/DataConverters/UInt8+Extensions.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  UInt8+Extensions.swift
+//  Purchases
+//
+//  Created by Andrés Boedo on 7/24/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+extension UInt8 {
```

Comment:
just conveniences to get a single bit's value and values in a given range in a byte